src/main/webapp/File/file.jelly

```diff
@@ -0,0 +1 @@
+<f:file field="hello" />
```

Comment:
Might be worth to ship a default for `accept`, I'd assume that's the most used restriction case.

Reply:
It’s in the docs, accept isn’t used very much, could add another example but not sure if it’s worth it

Reply:
Oic, can surely leave it as-is then 👍🏻